### PR TITLE
Exercise.005

### PR DESCRIPTION
**Pull request by mbonfiglio-mmul**
* Create Exercise.005.solutions

* Create Exercise.005

### CODE REVIEW

File: MMUL-BASH-01/Exercises/Day-1/Exercise.005 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Exercise 005 - substitutions with sed
```
Comment (rascasoft) [inline MMUL-BASH-01/Exercises/Day-1/Exercise.005]: First letter after - should be capital

File: MMUL-BASH-01/Exercises/Day-1/Exercise.005.solutions [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# Exercise 005 - substitutions with sed
+# Using the file results.txt of Excercise 004
+
+1) Substitute first match of 'root' with 'love'
+
```
Comment (rascasoft) [inline MMUL-BASH-01/Exercises/Day-1/Exercise.005.solutions:5]: Command prompt on all the other exercises solutions is:
```
[mmul@machine ~]$ 
```
Let's unify it.

**Review by rascasoft — CHANGES_REQUESTED** [review]
Few things need to be changed.

**Comment by mbonfiglio-mmul** [comment]
Fixed all issues.

**Review by rascasoft — APPROVED** [review]
Looks good to me!

**Pull request merged**